NDTensors/test/device_list.jl

```diff
@@ -23,9 +23,9 @@ function devices_list(test_args)
   end
 
   if "metal" in test_args || "all" in test_args
-    push!(devs, NDTensors.mtl)
+    push!(devs, mtl)
     Metal.allowscalar()
-    include(joinpath(pkgdir(NDTensors), "ext", "examples", "NDTensorMetal.jl"))
+    #include(joinpath(pkgdir(NDTensors), "ext", "examples", "NDTensorMetal.jl"))
```

Comment:
Why is this dropped?

Reply:
Sorry I commented it out while running tests on my machine, I'll add it back